app/src/main/java/org/p2p/wallet/send/model/Target.kt

```diff
@@ -50,4 +50,8 @@ data class Target constructor(
                 else -> Validation.INVALID
             }
         }
+
+    @IgnoredOnParcel
```

Comment:
I don't see reasons to add this attribute, can you describe why you add it here ?

Reply:
I've set this annotation similarly to previous properties and consider it just as utility property with no need to be parselized and transferred